docs/how-to/manage-users.md

```diff
@@ -0,0 +1,78 @@
+In this document, a user refers to a developer using fleet, not an end user accessing the magento site. Users are primarily used to manage authentication and authorization.
+
+Listing existing users
+----
+
+```
+$ fleet auth user list
+UUID                                 Email            Roles
+-----------------------------------  ---------------  -------
+4bb1538b-30f1-4017-9d82-6a5b3a407534 test@example.com Admin
+```
+
+The UUID is a unique identity generated for each user that never changes. However most actions accept an email as that is more readable.
+
+Creating a user
+----
+
+You can add [roles](/how-to/manage-roles) to a user when you create them, or you can do that separately. A user with no roles will not be authorized to do anything and has no authentication certificates.
```

Comment:
I this is a mental mistake on my part and I meant to write 'no authorization privileges'